### PR TITLE
Remove deprecated typealiases

### PR DESCRIPTION
**Pull request by wmcginty**
Remove the migration typealiases for types that were renamed in 4.0

### CODE REVIEW

**Review by br-tyler-milner — APPROVED** [review]

**Review by br-earl-gaspard — DISMISSED** [review]

**Comment by br-earl-gaspard** [comment]
Change log needed?

**Comment by wmcginty** [comment]
I figured no, because there isn't really any substantive changes here but I can add one anyway.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
```
Comment (br-earl-gaspard) [inline CHANGELOG.md:2]: Can you change to say `## Main` on line 1.

**Review by br-earl-gaspard — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (wmcginty) [reply CHANGELOG.md:2]: Done

**Review by br-earl-gaspard — APPROVED** [review]

**Review by br-tyler-milner — APPROVED** [review]

**Pull request merged**